### PR TITLE
report api calls latency

### PR DESCRIPTION
**Pull request by venkataanil**
1. Report api calls latency
2. Report max aggregated memory usage of masters, workers and infra nodes

### CODE REVIEW

File: cmd/kube-burner/ocp-config/metrics-report.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ---
-# Kubelet & CRI-O
+# API server
+- query: histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"LIST|GET", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope)) > 0
+  metricName: readOnlyAPICallsLatency
+
+- query: histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"POST|PUT|DELETE|PATCH", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope)) > 0
+  metricName: mutatingAPICallsLatency
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics-report.yml:7]: We should make these queries instant to be aligned with the others in this profile

Comment (venkataanil) [reply cmd/kube-burner/ocp-config/metrics-report.yml:7]: sure, thanks

File: cmd/kube-burner/ocp-config/metrics-report.yml [hunk 2]
```diff
@@ -1,5 +1,12 @@
 ---
-# Kubelet & CRI-O
+# API server
+- query: avg_over_time(histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"LIST|GET", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope))[{{.elapsed}}:]) > 0
+  metricName: readOnlyAPICallsLatency
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics-report.yml:4]: Just to maintain the format, can you rename the metrics to something like:
```
  metricName: avg-ro-apicalls-latency
``

File: cmd/kube-burner/ocp-config/metrics-report.yml [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+
+- query: avg_over_time(histogram_quantile(0.99, sum(irate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb=~"POST|PUT|DELETE|PATCH", subresource!~"log|exec|portforward|attach|proxy"}[2m])) by (le, resource, verb, scope))[{{.elapsed}}:]) > 0
+  metricName: mutatingAPICallsLatency
```
Comment (rsevilla87) [inline cmd/kube-burner/ocp-config/metrics-report.yml:7]: same as above

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**